wisp-logging/src/test/kotlin/wisp/logging/LoggingTest.kt

```diff
@@ -9,18 +9,16 @@ import org.junit.jupiter.api.Test
 import org.slf4j.MDC
 
 class LoggingTest {
-    var wispQueuedLogCollector = WispQueuedLogCollector()
-
-    private var logCollector: LogCollector = wispQueuedLogCollector
+    private val logCollector = WispQueuedLogCollector()
```

Comment:
Some small clean up. I didn't find the extra variable valuable and a bit confusing to read, but happy to revert if others prefer it this way.

Reply:
Might have been a legacy during refactoring a long time ago and not cleaned up.  LGTM.